docs/DeveloperGuide.md

```diff
@@ -1,34 +1,245 @@
+
+Figure below shows the sequence diagram of `SummaryCommand` class.
+
+![Sequence Diagram of SummaryCommand class](images/summary.png) </br>
+Figure 8: Sequence diagram of `SummaryCommand` class
+
+### 3.8 Reminder Feature
+`SpendingList` and `WarnCommand` facilitate this feature. The Reminder feature is able to provide the user about the 
+total expenditure of the current week, starting on Monday. It implements the following operations:
+* `SpendingList#getSpendingAmountTime(period)` 
+* `WarnCommand#execute(spendingList, ui)`
+
+Below shows an example of usage:
+
+1. User starts the application.
+2. The `Reminder` will be instantiated. The dates of the current week (starting from Monday) will be saved to a list.
+3. In the `Reminder#execute(spendinglist, ui)`, a check will be done to see if there is any budget being set by the user.
+    * If no budget is being set, the total expenditure of current week will be tallied up. 
+    * If there is, `WarnCommand#execute(spendingList, ui)` will be called first before tallying up the expenditure. 
+
+Figure below shows the sequence diagram of Reminder class.
+![Sequence Diagram of SummaryCommand class](images/reminder.png) </br>
+Figure 9: Sequence diagram of `SummaryCommand` class
+
+### 3.9 Export Feature
+`Workbook`, `FileOutputStream`, `SpendingList` and `Ui` facilitate this feature. The export data feature could extract the current data and export to an Excel file. It implements the following operations:
+* `Workbook#createSheet()` → creates a sheet in the workbook
+* `SpendingList#getItem()` → gets the item waiting to be added to the workbook
+* `Workbook#write()` → writes the data to an Excel file
+* `Ui#printExportMessage()` → prints the message indicating that the Excel file has been created successfully
+
+Below shows an example of usage:
+
+1. User executes the `export F:\` command to export the data to the location `F:\`.
+2. The `export` command calls the `Workbook#createSheet()` to create an Excel workbook sheet.
+3. `SpendingList#getItem()` will be called repeatedly to get items and set the cells' values accordingly.
+3. After writing to an Excel file at the location `F:\` using `Workbook#write()`, a prompt message will be shown by `Ui`.
+
+Figure below shows the sequence diagram of `ExportCommand` class.
+![Sequence Diagram of ExportCommand class](images/ExportCommand.png) </br>
+Figure 10: Sequence diagram of `ExportCommand` class
+
+
+## 4. Product scope
+### 4.1 Target user profile
+
+The target user is NUS hostel students including both local students and international students.
+
+### 4.2 Value proposition
+
+We observed that hostel students are encountering tight budgets due to expensive hostel fees, on top of their school 
+tuition fees. Hence, we are developing a money manager targeted at NUS hostel students. This money manager   
+Cent Wise Dollar Wise aims to help hostel students to have better financial management with limited budget. 
+
```

Comment:
![image](https://user-images.githubusercontent.com/56764814/97660163-ba2a5b80-1aab-11eb-8bad-ed8053e0876c.png)


There is a weird new line over here, might want to look into it